From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mack Straight <mack@discordapp.com>
Date: Wed, 5 Feb 2020 17:59:26 -0800
Subject: Use SW_SHOW for initial state, not SHOWNORMAL.

This allows ui::SHOW_STATE_DEFAULT to be used to show a window without
altering its tiling or maximize state.

diff --git a/ui/views/widget/desktop_aura/desktop_window_tree_host_win.cc b/ui/views/widget/desktop_aura/desktop_window_tree_host_win.cc
index 9949d73b85b5110711689c5ee3f8ba1a493cc976..540ca59a1fd02006f494857e1fb74e207cee366c 100644
--- a/ui/views/widget/desktop_aura/desktop_window_tree_host_win.cc
+++ b/ui/views/widget/desktop_aura/desktop_window_tree_host_win.cc
@@ -853,7 +853,7 @@ bool DesktopWindowTreeHostWin::IsModal() const {
 }
 
 int DesktopWindowTreeHostWin::GetInitialShowState() const {
-  return CanActivate() ? SW_SHOWNORMAL : SW_SHOWNOACTIVATE;
+  return CanActivate() ? SW_SHOW : SW_SHOWNA;
 }
 
 int DesktopWindowTreeHostWin::GetNonClientComponent(
